views/room.ejs

```diff
@@ -400,6 +403,80 @@
       ga('send', 'pageview');
 
     </script>
-
+    <% if (adobeTrackingUrl) { %>
+    <script type="text/javascript">
+      var digitalData = {
+        page: {
+          pageInfo: {
+            pageName: '',
+            lob: 'biz',
+            functionDept: '<%=ATFunctionDept%>',
+            subCategory1: '',
+            subCategory2:'',
+            subCategory3: '',
+            primaryCategory: '<%=ATPrimaryCategory%>',
+            internalSearchTerm: '',
+            internalSearchResults: '',
+            error: '',
+            pageType: '',
+            language: '<%=userLanguage%>',
+            country: '<%=userCountry%>',
+            fileGatedType: '',
+            tollFreeNumber: '',
+            siteIdentifier: '<%=ATSiteIdentifier%>',
+            type: '',
+            pageCategory: '',
+            pageSubCategory: '',
+          },
+          content: {
+            name: '',
+            category: '',
+            author: '',
+            type: '',
+            publishDate: '',
+            subCategory: '',
+            topic: '',
+            industry: '',
+            businessSize: '',
+            profRole: '',
+            company: '',
+            product: '',
+          },
+        },
+        user: {
+          profile: {
+            profileInfo: {
+              targetExperience: '',
+              responsiveSiteVersion: 'desktop',
+              visitorType: 'prospect',
+              gaCampaign: '',
+            },
+          },
+        },
+        transaction: {
+          item: {
+            quantity: '',
+            price: '',
+            productInfo: '',
+          },
+        },
+        eventData: {
+          fIndustry: '',
+          fName: '',
+          fCTA: '',
+          fLines: '',
+          linesSlider: '',
+          interactionType: '',
+          events: '',
+        },
+      };
+      (function(){
+        var width = window.screen.width;
+        var responsive = ( width < 768 ) ? "mobile" : ( width < 960 ) ? "tablet" : "desktop";
+        digitalData.user.profile.profileInfo.responsiveSiteVersion = responsive;
+      })();
+      if (typeof _satellite !== "undefined") _satellite.pageBottom();
```

Comment:
Same comment as above: can we use `if (_statellite) _satellite.pageBottom()` (or `_satellite && _satellite.pageBottom()`)?

Reply:
@jeffswartz I didn't want to change the code provided by the Adobe team:
<img width="578" alt="Screen Shot 2020-04-13 at 13 47 00" src="https://user-images.githubusercontent.com/12943780/79154915-9280d980-7d8d-11ea-9f83-e9e9d80c95e2.png">

What do you think? Should I change it?

Reply:
No, it's fine.